BlackNectar/BlackNectar.xcodeproj/project.pbxproj

```diff
@@ -351,7 +370,7 @@
 				TargetAttributes = {
 					321319391DE0F61500FB1774 = {
 						CreatedOnToolsVersion = 8.1;
-						DevelopmentTeam = 3TAADZBCK9;
+						DevelopmentTeam = 5E82HJH2N5;
```

Comment:
Once again, watch the development team. We shouldn't be changing this with each PR. We need to set it to none.